host/3.0/buster/amd64/base.Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS runtime-image
 
 ENV PublishWithAspNetCoreTargetManifest=false
-ENV HOST_VERSION=3.0.12915
+ENV HOST_VERSION=3.0.12930
 ENV HOST_COMMIT=a15c411a38931de0457764a19f22eb9fa0edeefc
```

Comment:
you'll need to update the commit as well. We need to do a similar change to https://github.com/Azure/azure-functions-docker/commit/8cd18aa847956e26950f9c8afa127d91a1f64f5f for v3